packages/core/core/src/services/document-service/entries.ts

```diff
@@ -0,0 +1,57 @@
+import type { UID } from '@strapi/types';
+
+import * as components from './components';
+
+import { transformParamsDocumentId } from './transform/id-transform';
+import { transformParamsToQuery } from './transform/query';
+import { pickSelectionParams } from './params';
+import { applyTransforms } from './attributes';
+import entityValidator from '../entity-validator';
+
+const createEntriesService = (uid: UID.ContentType) => {
```

Comment:
does it make sense to also expose an update entry?

Reply:
I'm pushing that in a minute :)